### ISSUE TITLE
Feature Request: When account email address is updated, offer to update WHOIS email

### ISSUE THREAD

**Issue by supernovia** [Automattic/wp-calypso#95673]
### What

When a user updates their email address at https://wordpress.com/me/account, we could have a toggle or checkbox for updating the domain contact email as well.

### Why

Users who own domains sometimes will update their account email but forget to update their domain's whois email, which can lead to a suspension for lack of verification.

### How

There are a few ways we could approach this depending on how complex we wanted to be.

eg: We could _remind_ people to update domain emails and give them a link. This would have to be dismissable

or: We could _offer_ to update emails for domains, with a checkbox with each domain and its old contact email. We shouldn't update emails automatically, though, in case they have some other email intentionally.

**Comment by inaikem** [comment]
This is a GREAT idea, Velda!

@Automattic/yolo, I've assinged this to you based on the current product are walkthrough divisions. Please reassign it to another team as you see fit!

**Comment by wojtekn** [comment]
@inaikem would @Automattic/nomado be a better fit?

**Comment by ramynasr** [comment]
@wojtekn, thanks for the ping!

I'm happy to help with providing any details necessary for the team who will be working on this. Nomado can definitely help shape the specs related to this issue to make sure we stay compliant. The actual UI work required for that can be picked up by any team with the bandwidth for it.

**Comment by inaikem** [comment]
> @/inaikem would @/Automattic/nomado be a better fit?

@wojtekn 👋  I tihnk Nomado is a historically great fit. That said, I pinged Yolo because of the current walkthrough focus. I'm not married to it having to land with a specific team as long as the FR is considered and prioritised appropriately!

**Comment by katinthehatsite** [comment]
I think we would need some design decisions / work on this. 

One thing to keep in mind that it is possible to have a different email address for WHOIS and for your WP.com account (I have my own domain as an example use case). Thus, I do not think it is a good approach to update the email automatically for the user for WHOIS to avoid making unwanted changes. We could possibly add a reminder or notice for the user to double-check their email addresses for WHOIS though.

@matt-west I am not sure if you are working on Calypso now or not, would you be able to take a look? 
If you are not the right person to ping, do you know whom I can reach out to? Thank you!

**Comment by matt-west** [comment]
@katinthehatsite I don’t have capacity to pick this up at the moment, but will ask if another designer can work on it.

**Comment by wojtekn** [comment]
@keoshi, what if we made it simple for a start, added a notice with a link, and tracked the number of users who use it?

> eg: We could remind people to update domain emails and give them a link. This would have to be dismissable

**Comment by keoshi** [comment]
Yeah that's great, @wojtekn. As long as we're tracking the usage/interest in some way I see that as moving forward. We could start very MVP and build more and more robust solutions if we see a trend that justifies it.

**Comment by katinthehatsite** [comment]
Thanks for taking a look @keoshi  - any specific notice / component you have in mind? Should the link send the user to https://wordpress.com/domains/manage or any other page? 🙇

**Comment by keoshi** [comment]
@katinthehatsite If the goal is to reuse Core components, would suggest this one: https://wordpress.github.io/gutenberg/?path=/docs/components-notice--docs

If we plan to have an intermediary solution in Calypso, this seems to be the component to do it with: https://wpcalypso.wordpress.com/devdocs/design/notice

As for the final URL, would send to that page if there's more than one domain. If there's only one they manage, would be good to send them directly to https://wordpress.com/domains/manage/EXAMPLE.com/edit-contact-info/EXAMPLE.COM

**Comment by katinthehatsite** [comment]
Sounds good, thanks for taking a look @keoshi - I will proceed with implementation then 🙇

**Comment by keoshi** [comment]
Thank you so much, @katinthehatsite! 🙌

**Comment by katinthehatsite** [comment]
@keoshi @wojtekn I gave it a go here: https://github.com/Automattic/wp-calypso/pull/98586 - feel free to add recommendations for the wording. I did not add tracking yet as we will need a new event for it. I will add it in a follow-up PR 🙇

**Comment by katinthehatsite** [comment]
To sum up, for now we implemented a more precise and descriptive message for updating the domain email after in the notice that appears after the user updates their email address. We have also added tracks event to see how this feature is used. We can revisit this in the future and see whether further improvements are needed.
